tendrl/node_agent/monitoring/collectd/collectors/gluster/low_weight/tendrl_glusterfs_brick_disk_stats.py

```diff
@@ -0,0 +1,421 @@
+import psutil
+import socket
+import threading
+import time
+import traceback
+
+
+import collectd
+from tendrl_gluster import TendrlGlusterfsMonitoringBase
+import utils as gluster_utils
+
+
+class TendrlBrickDeviceStatsPlugin(
+    TendrlGlusterfsMonitoringBase
+):
+    def __init__(self):
+        self.provisioner_only_plugin = False
+        TendrlGlusterfsMonitoringBase.__init__(self)
+        self.STAT_INTERVAL_FOR_PER_SEC_COUNTER = 10
+        self.brick_details = {}
+
+    def get_brick_source_and_mount(self, brick_path):
+        # source and target correspond to fields "Filesystem" and
+        # "Mounted on" from df command output. The below command
+        # gives the filesystem and mount point for a given path,
+        # Eg: "/dev/mapper/tendrlMyBrick4_vg-tendrlMyBrick4_lv " \
+        #     "/tendrl_gluster_bricks/MyBrick4_mount"
+
+        command = "df --output=source,target " + brick_path
+        out, err = gluster_utils.exec_command(command)
```

Comment:
Cant we use `cmd_utils` from commons here?

Reply:
I thought about it but that would make the plugins tied to tendrl but now except for config the plugins are independent and don't have any dependency on any tendrl module

Reply:
Ack